Vagrantfile

```diff
@@ -16,7 +16,7 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
   # please see the online documentation at vagrantup.com.
 
   config.vm.provider :virtualbox do |vbox|
-    config.vm.box = "ubuntu/xenial64"
+    config.vm.box = "ubuntu/bionic64"
```

Comment:
Is there a way we can configure this? It would be nice to test both Ubuntu versions.

Reply:
You can change that line at any time, and do `$ vagrant destroy && vagrant up`. I could look at having multiple Vagrant setups, but does anyone else use it much?